MEAnalysis/python/MEAnalysis_heppy.py

```diff
@@ -279,6 +279,7 @@ def prepareInputSamples(sampleFile=conf.general["sampleFile"]):
                 xs = s.xSec.value()
             )
             inputSample.isMC = s.isMC.value()
+            inputSample.skip = s.skip.value() #DS
```

Comment:
We've removed the "skip" parameter as the grid-control way of submitting the samples already took care of it in a more simple way.